docs/README.md

```diff
@@ -26,13 +27,14 @@ Theme is copied from Daml Docs theme.
 
 ### Sass
 
-The `ruby` version of `sass` must be installed: `gem install sass`.
+The `ruby` version of `sass` must be installed: `sudo gem install sass`.
 
 ### Grunt
 
-If grunt is not already installed: `npm install grunt-cli`.
+If grunt is not already installed: `sudo npm install -g grunt-cli`.
```

Comment:
You had to install `grunt-cli` globally ?

Reply:
as discussed, yes